docs/content/docs/testing/devtools.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Fluid Developer Tools (DevTools)
+menuPosition: 4
+aliases:
+  - "/docs/start/devtools"
+---
+
+As Fluid Framework abstracts away the complexities of dealing with real-time collaborative data, it is crucial that developers have access to the underlying state when debugging their Fluid-based applications. **Fluid Developer Tools (Fluid DevTools)** is a browser extension that lets developers debug their Fluid-based applications. It is built as an extension of the browser developer tools that is already part of many developers' workflows. It is open-source, extensible and community contributions are welcome at https://aka.ms/fluid/devtool. The Fluid DevTools provides access to the following features:
+
+-   Container information and data visualization
+-   Ability to modify container state for testing offline and reconnect scenarios
+-   Visibility into audience membership, permissions, and join/leave logs
+-   Framework event logs to view performance and error events
+
+To use the Fluid DevTools, developers need to follow two simple steps:
+
+1. Integrate the Fluid DevTools library into your app using these steps: https://aka.ms/fluid/devtool
+    -   See DevTools integrated into samples at https://aka.ms/fluid/samples
+2. Install the Fluid DevTools browser extension from [Edge](https://aka.ms/fluid/devtool/edge) or [Chrome](https://aka.ms/fluid/devtool/chrome) extension stores
+
+Once you have completed the above steps, you can simply launch your Fluid application in a browser window and start the browser DevTools by right-clicking and selecting `Inspect`. In the browser DevTools window, you should see a tab for `Fluid Developer Tools`
+
+## Container Information
+
+DevTool shows all the containers in your application, their current state (connected, attached, disconnected) and allows developers to modify the state (disconnect, close). Developers can also see a lot of container state changes with timestamps.
```

Comment:
Nit. It's probably an uncommon scenario that someone would _not_ want to pass a particular container to Devtools, but I think it's worth it to call out the need for user action here (and it's probably spelled out in more detail in the docs linked further up); otherwise it sounded a bit auto-magical to me.
```suggestion
DevTool shows all the containers from your application that you tell it about, their current state (connected, attached, disconnected) and allows developers to modify the state (disconnect, close). Developers can also see a lot of container state changes with timestamps.
```

Reply:
Fair point, will update to say only containers specified to the devtools will show here